### DISCUSSION TITLE
Version 1.30.0

### DISCUSSION THREAD

**Discussion by apple502j** [ScratchAddons/ScratchAddons#5548] [Changelog]
## Bugfixes
- [#5208](https://github.com/ScratchAddons/ScratchAddons/issues/5208) Preset loading message does not mention that it overwrites current setting. (prio-4)
- [#5231](https://github.com/ScratchAddons/ScratchAddons/issues/5231) rename-broadcasts refers to messages as "broadcasts". (prio-4)
- [#5260](https://github.com/ScratchAddons/ScratchAddons/issues/5260) hide-flyout speed settings are unclear. (prio-4)
- [#5267](https://github.com/ScratchAddons/ScratchAddons/issues/5267) Setting pages are not RTL-ready. (prio-4)
- [#5278](https://github.com/ScratchAddons/ScratchAddons/issues/5278) "Scratch Addons" remains unlocalized in browser action titles. (prio-4)
- [#5283](https://github.com/ScratchAddons/ScratchAddons/issues/5283) pick-colors-from-stage is incompatible with opacity-slider. (prio-3)
- [#5285](https://github.com/ScratchAddons/ScratchAddons/issues/5285) block-duplicate does not follow the block-cherry-picking setting. (prio-4)
- [#5302](https://github.com/ScratchAddons/ScratchAddons/issues/5302) dark-www makes profile report modal unreadable. (prio-4)
- [#5323](https://github.com/ScratchAddons/ScratchAddons/issues/5323) scratch-messaging crashes when the profile is deleted. (prio-2)
- [#5338](https://github.com/ScratchAddons/ScratchAddons/issues/5338) custom-zoom speed settings are unclear. (prio-4)
- [#5351](https://github.com/ScratchAddons/ScratchAddons/issues/5351) editor-theme3 and scratchblocks do not apply to some forum pages. (prio-3)
- [#5352](https://github.com/ScratchAddons/ScratchAddons/issues/5352) comments-linebreaks setting name is unclear. (prio-4)
- [#5362](https://github.com/ScratchAddons/ScratchAddons/issues/5362) dark-www does not apply to the new Scratch Link page. (prio-3)
- [#5366](https://github.com/ScratchAddons/ScratchAddons/issues/5366) my-ocular status can overflow in the forums. (prio-4)
- [#5368](https://github.com/ScratchAddons/ScratchAddons/issues/5368) material-forum-editor-buttons does not work with image-uploader. (prio-4)
- [#5372](https://github.com/ScratchAddons/ScratchAddons/issues/5372) turbowarp-player reuses an expired token. (prio-2)
- [#5377](https://github.com/ScratchAddons/ScratchAddons/issues/5377) debugger does not translate addon block names inside the threads tab. (prio-4)
- [#5379](https://github.com/ScratchAddons/ScratchAddons/issues/5379) debugger does pause the sound when resumed then immediately paused. (prio-4)
- [#5392](https://github.com/ScratchAddons/ScratchAddons/issues/5392) discuss-button compact dropdown feature sometimes does not work. (prio-4)
- [#5392](https://github.com/ScratchAddons/ScratchAddons/issues/5392) discuss-button compact dropdown has extra space. (prio-4)
- [#5395](https://github.com/ScratchAddons/ScratchAddons/issues/5395) Addons run in Contact Us form in Firefox only. (prio-4)
- [#5405](https://github.com/ScratchAddons/ScratchAddons/issues/5405) vol-slider resets when entering the editor. (prio-3)
- [#5405](https://github.com/ScratchAddons/ScratchAddons/issues/5405) mute-project sets the volume to default and not the former value when unmuting. (prio-3)
- [#5405](https://github.com/ScratchAddons/ScratchAddons/issues/5405) mute-project sets the volume to 100% and not the default when disabled. (prio-4)
- [#5405](https://github.com/ScratchAddons/ScratchAddons/issues/5405) mute-project icon is selectable. (prio-4)
- [#5405](https://github.com/ScratchAddons/ScratchAddons/issues/5405) vol-slider muted indicator is not visible in small stage mode. (prio-4)
- [#5424](https://github.com/ScratchAddons/ScratchAddons/issues/5424) hide-flyout is incompatible with sprite-folders. (prio-3)
- [#5426](https://github.com/ScratchAddons/ScratchAddons/issues/5426) better-img-uploads does not add the button to backdrop chooser. (prio-3)
- [#5428](https://github.com/ScratchAddons/ScratchAddons/issues/5428) scratch-messaging does not style Scratch Team messages properly. (prio-4)
- [#5429](https://github.com/ScratchAddons/ScratchAddons/issues/5429) Scratch Addons conflicts with Redux DevTools extension. (prio-3)
- [#5446](https://github.com/ScratchAddons/ScratchAddons/issues/5446) turbowarp-player has a white background in dark mode while in fullscreen. (prio-4)
- [#5447](https://github.com/ScratchAddons/ScratchAddons/issues/5447) turbowarp-player has a white background in dark mode. (prio-4)
- [#5468](https://github.com/ScratchAddons/ScratchAddons/issues/5468) find-bar crashes when an expression is inside broadcast blocks. (prio-2)
- [#5468](https://github.com/ScratchAddons/ScratchAddons/issues/5468) find-bar crashes with object prototype attribute names as broadcast names. (prio-2)
- [#5491](https://github.com/ScratchAddons/ScratchAddons/issues/5491) curator-link does not work when logged out. (prio-4)
- [#5492](https://github.com/ScratchAddons/ScratchAddons/issues/5492) editor-stage-left flips addon icons. (prio-4)
- [#5496](https://github.com/ScratchAddons/ScratchAddons/issues/5496) onion-skinning labels wrap in unexpected ways. (prio-4)
- [#5498](https://github.com/ScratchAddons/ScratchAddons/issues/5498) fullscreen does not resize the stage canvas. (prio-3)
- [#5499](https://github.com/ScratchAddons/ScratchAddons/issues/5499) fullscreen positions the answer input at incorrect location. (prio-4)
- [#5500](https://github.com/ScratchAddons/ScratchAddons/issues/5500) hi-res-thumbnails applies to forum posts. (prio-4)
- [#5500](https://github.com/ScratchAddons/ScratchAddons/issues/5500) hi-res-thumbnails breaks on dynamic disable. (prio-4)
- [#5505](https://github.com/ScratchAddons/ScratchAddons/issues/5505) editor-dark-mode makes the paint editor caret invisible. (prio-4)
- [#5507](https://github.com/ScratchAddons/ScratchAddons/issues/5507) Addons run at email pages. (prio-4)
- [#5509](https://github.com/ScratchAddons/ScratchAddons/issues/5509) editor-dark-mode makes project title and remix button unreadable. (prio-4)
- [#5510](https://github.com/ScratchAddons/ScratchAddons/issues/5510) find-bar does not scroll dropdowns when using keyboard navigation. (prio-4)
- [#5512](https://github.com/ScratchAddons/ScratchAddons/issues/5512) middle-click-popup is incompatible with rename-broadcasts. (prio-3)
- [#5522](https://github.com/ScratchAddons/ScratchAddons/issues/5522) A girl is referred to as a dude. (prio-1)
- [#5535](https://github.com/ScratchAddons/ScratchAddons/issues/5535) find-bar adds another find bar in variable-manager tab. (prio-4)
- [#5535](https://github.com/ScratchAddons/ScratchAddons/issues/5535) find-bar label text does not focus the input when clicked. (prio-4)
- [#5535](https://github.com/ScratchAddons/ScratchAddons/issues/5535) find-bar shortcuts do not work after reentering the editor. (prio-4)
- [#5536](https://github.com/ScratchAddons/ScratchAddons/issues/5536) hide-delete-button does not hide the delete sound button. (prio-3)
- [#5537](https://github.com/ScratchAddons/ScratchAddons/issues/5537) variable-manager does not prevent conflicting names from being used. (prio-3)
- [#5542](https://github.com/ScratchAddons/ScratchAddons/issues/5542) editor-theme3 does not refresh the toolbox when it should. (prio-3)

## New Features
- [#5006](https://github.com/ScratchAddons/ScratchAddons/issues/5006) New addon (split from DevTools addon): middle-click-popup (prio-3)
- [#5059](https://github.com/ScratchAddons/ScratchAddons/issues/5059) New addon: paint-snap (prio-3)
- [#5158](https://github.com/ScratchAddons/ScratchAddons/issues/5158) New addon: sprite-properties (prio-3)
- [#5399](https://github.com/ScratchAddons/ScratchAddons/issues/5399) New addon: number-pad (prio-3)

## Changes
- [#5333](https://github.com/ScratchAddons/ScratchAddons/issues/5333) block-switching: always enable borders (prio-3)
- [#5378](https://github.com/ScratchAddons/ScratchAddons/issues/5378) editor-extra-keys: rename addon to "Extra key options" (prio-3)
- [#5413](https://github.com/ScratchAddons/ScratchAddons/issues/5413) 60fps is no longer enabled by default for new users. (prio-3)
- [#5413](https://github.com/ScratchAddons/ScratchAddons/issues/5413) pause is now enabled by default for new users. (prio-3)
- [#5496](https://github.com/ScratchAddons/ScratchAddons/issues/5496) onion-skinning now applies background color when setting input is focused. (prio-3)

**Comment by Samq64** [comment]
50+ bugs fixed!

**Comment by CST1229** [comment]
> * [#5522](https://github.com/ScratchAddons/ScratchAddons/issues/5522) A girl is referred to as a dude. (prio-1)

Is this having priority 1 actually serious? (Not trying to come off as disrespectful or anything, just asking.)

**Reply by Samq64** [reply]
Yeah, I'm pretty sure similar PRs got priority 4 and I don't think what the username changed to should make much of a difference.

**Reply by WorldLanguages** [reply]
It doesn't really matter whether we consider it strictly prio-1 or not, but I'd say it's one of those PRs that we'd merge as soon as possible.

**Reply by Samq64** [reply]
Yeah, almost all hotfixes are merged within a few hours anyway.
